src/XrmMockupShared/Core.cs

```diff
@@ -604,6 +607,25 @@ internal OrganizationResponse Execute(OrganizationRequest request, EntityReferen
             return response;
         }
 
+        private void CopySystemAttributes(Entity postImage, Entity item1)
+        {
+            if (item1 == null)
+            {
+                return;
+            }
+
+            item1["createdon"] = postImage.GetAttributeValue<DateTime>("createdon");
```

Comment:
The code handles these 3 attributes in the same way. Can we maybe generalize? Maybe move the attributes to a list that the code iterates over and adds them to the item1. This will also make it easier to extend with additional attributes. Both me and @magesoe has a suspicion that there are other system attributes we need to do this with as well.

Reply:
I've made the solution more generic, with a list of system attribute names which is iterated over.

Reply:
Could #92 be resolved via this as well ?